src/commands/gist.ts

```diff
@@ -402,6 +402,43 @@ export async function registerGistCommands(context: ExtensionContext) {
     )
   );
 
+  context.subscriptions.push(
+    commands.registerCommand(
+      `${EXTENSION_NAME}.viewForkedGist`,
+      async (node?: StarredGistNode | FollowedUserGistNode) => {
+        await ensureAuthenticated();
+
+        if (node) {
+          const forks = await getForks(node.gist.id);
+
+          const items = sortGists(forks).map((g) => ({
+            label: getGistLabel(g),
+            description: getGistDescription(g),
+            owner: g.owner.login,
+            id: g.id
+          }));
+
+          if (items.length === 0) {
+            const message = `Oops, this Gist doesn't have any forks.`;
+            return window.showInformationMessage(message);
+          }
+
+          const selected = await window.showQuickPick(items, {
+            placeHolder: "Select the forked Gist you'd like to open...",
+            matchOnDescription: true
+          });
+
+          if (selected) {
+            openGistFiles(selected.id);
+          }
+        } else {
+          const message = `Oops, there's an error finding the Gist.`;
```

Comment:
If you want, you can probably remove this `else` clause, since the command is only exposed from the tree view, and therefore, wouldn't be callable without `node` being defined.